20231019/examples/invalid/conceptWithMonolingualLabels.json

```diff
@@ -0,0 +1,61 @@
+{
+  "@context": [
+    "https://w3id.org/kim/amb/context.jsonld",
+    {
+      "@language": "de"
+    }
+  ],
+  "name": "Arbeitsblatt - Mon avenir - Französisch - tutory.de",
+  "id": "https://www.tutory.de/w/fbbadf1a",
+  "image": "https://www.tutory.de/worksheet/fbbadf1a-145a-463d-9a43-1ae9965c86b9.jpg?width=1000",
+  "type": ["LearningResource"],
+  "creator": [
+    {
+      "name": "HerunterS",
+      "type": "Person"
+    }
+  ],
+  "description": "Französisch-Arbeitsblatt",
+  "about": [
+    {
+      "id": "https://w3id.org/kim/hochschulfaechersystematik/n059",
+      "prefLabel": "Französisch",
+      "type": "Concept",
+      "inScheme": {
+        "id": "https://w3id.org/kim/hochschulfaechersystematik/scheme"
+      }
```

Comment:
`inScheme` entfernen, da es auch in der Spec nicht mehr erwähnt wird?

Reply:
Fixed with [39adb10](https://github.com/dini-ag-kim/amb/pull/256/commits/39adb101cd746293be87af597cd4f2262f3e622e)